easybuild/easyconfigs/a/AOCC/AOCC-3.1.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,27 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'AOCC'
+version = '3.1.0'
+
+homepage = 'https://developer.amd.com/amd-aocc/'
+description = "AMD Optimized C/C++ & Fortran compilers (AOCC) based on LLVM 12.0"
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['http://developer.amd.com/wordpress/media/files/']
```

Comment:
@robert-mijakovic maybe worth adding a comment here about the EULA, to be consistent with other similar easyconfigs?

Reply:
Done